griptape/drivers/vector/base_vector_store_driver.py

```diff
@@ -10,14 +10,14 @@
 
 from griptape import utils
 from griptape.artifacts import BaseArtifact, ListArtifact, TextArtifact
-from griptape.mixins import SerializableMixin
+from griptape.mixins import EventsMixin, SerializableMixin
 
 if TYPE_CHECKING:
     from griptape.drivers import BaseEmbeddingDriver
 
 
 @define
-class BaseVectorStoreDriver(SerializableMixin, ABC):
```

Comment:
Should there be a `BaseDriver` class that has the common stuff for all drivers, like `EventsMixin` and `SerializableMixin`? Or are there exceptions to this?

Reply:
Yeah probably. I've got an item to standardize all the Driver interfaces pre 1.0. So let's do that then.